main.py

```
@@ -20,23 +25,81 @@ def health():
+#     inputs = {yollo_session.get_inputs()[0].name: input_tensor}
+#     outputs = yollo_session.run(None, inputs)
+
+#     # 후처리 (예: 바운딩 박스, 클래스 라벨)
+#     result = process_outputs(outputs)
+
+#     return {"predictions": result}
+
+
+@app.post("/clip/embedding")
+async def clip_embed(
+    image: Optional[UploadFile] = File(None), url: Optional[str] = Form(None)
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)